cv.tex

```diff
@@ -150,102 +149,70 @@ \subsection{Programming}
 
 \section{Experience}
 
-\runsubsection{Coursera}
-\descript{| KPCB Fellow + Software Engineering Intern}
+\runsubsection{Dropbox}
```

Comment:
(JC):
- Spell out November in both cases for consistency with rest of document.
- Looks like you’re using past tense. Start your sentence with “Fulfilled Lead Incident Response role for the….”
- Probably don’t include the word “malicious” in the title of a hyperlink. Maybe “Data Sharing” or something like that.
- Consider putting the Dropbox Sign link first as that seems more interesting of the two.
- Do you need to put numbers in the link titles? As long as they are visually distinguishing I think you’re find without numbers.
- “Lead emergent issues” —> “Led emergent issues”
- "Lead emergent issues with Audit, Compliance, Risk teams, documenting playbooks and interaction outcomes” —> “Led emergent issues with Audit, Compliance, and Risk teams; documented playbooks and engagement outcomes”
- “Mentored several great security engineers” —> “Mentored security engineers” or “Mentored N security engineers”
- “Reduction in Force” —> “reduction in force”

Reply:
1. ty
2. ty
3. ty, small edits but leaving the naughty word
4. agree, leaving them in chronological order tho
5. ty
6-7. ty with slight mods
8. ty
9. ty